File: programmers/고득점 Kit/[탐욕법]-구명보트.py
def solution(people, limit):
    answer,i,j = 0, 0, len(people)-1
    people.sort()
    while i<j:
        if people[i]+people[j] <= limit: i,j,answer = i+1,j-1,answer+1
        else: j-=1
    return len(people)-answer

File: programmers/고득점 Kit/javascript/[탐욕법] 구명보트.js
function solution(people, limit) {
    let answer=0, left=0, right=people.length-1
            people.sort((a,b)=>a-b)
            while(left<right){
                if(people[left]+people[right]<=limit){
                    answer++
                    left++
                    right--
                }
                else right--
            }
            return people.length-answer
}